package.json

```diff
@@ -11,7 +11,8 @@
     "start-logging": "node app.js | ./node_modules/.bin/bunyan",
     "lint": "eslint --ext .js,.json .",
     "test": "NODE_ENV=test mocha --recursive test",
-    "watch-dev": "nodemon app.js | ./node_modules/.bin/bunyan",
+    "build-css": "./css-build",
+    "watch-dev": "nodemon app.js | ./node_modules/.bin/bunyan | ./css-watch",
```

Comment:
If you use a single ampersand rather than a pipe both processes will run and you will see output from them in the foreground. You can also get rid of the `css-watch` file and do it all in one command e.g. `nodemon app.js | ./node_modules/.bin/bunyan & nodemon -e scss -x npm run build-css`,